SDWebImage/SDWebImageDownloaderOperation.h

```diff
@@ -66,6 +66,8 @@ FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadFinishNotification
  */
 @property (nonatomic, strong, nullable) NSURLCredential *credential;
 
+@property (nonatomic, copy, nullable) Class internalCoderClass;
```

Comment:
Fixed by use just an image coder for each `SDWebImageDownloaderOperation`. It will be set when the `SDWebImageDownloader` created the operation. If you do not set one, create an built-in decoder instance. (each operation need a seperate coder to avoid data conflicts)